### PR TITLE
[10.0][BACKPORT] account_move_chatter backport from 12.0

### PR DESCRIPTION
**Pull request by bodedra**
Standard backport module + track JE's status.

### CODE REVIEW

**Comment by bodedra** [comment]
@max3903 @dreispt

**Comment by bodedra** [comment]
Travis fails but not related with this PR.

**Review by max3903 — APPROVED** [review]

File: account_move_chatter/models/account_move.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
-from odoo import models
+from odoo import models, fields
 
 
 class AccountMove(models.Model):
-    _inherit = ["account.move", "mail.thread", "mail.activity.mixin"]
+    _inherit = ["account.move", "mail.thread"]
     _name = "account.move"
+
+    state = fields.Selection(track_visibility='always')
```
Comment (pedrobaeza) [inline account_move_chatter/models/account_move.py:12]: This is something unrelated  to the purpose of the module and shouldn't be here IMO.

**Review by pedrobaeza — DISMISSED** [review]

File: account_move_chatter/models/account_move.py (hunk 1 above)
Comment (bodedra) [reply account_move_chatter/models/account_move.py:12]: Purpose of this change is to track user who is responsible for create, validate, cancel that accounting entry. In v12, this feature covered but in v10, we have to add `track_visibility` attributes. So I did migration and improve little bit.

If you don't like this improvements in a migration PR, I will rollback this changes. And then I will propose a new PR for enhancement. LMK your suggestion.

Thank you.

Comment (pedrobaeza) [reply account_move_chatter/models/account_move.py:12]: Ok, let's keep it if the final purpose is to include it in the module.

**Comment by pedrobaeza** [comment]
Please check Travis

**Review by dreispt — APPROVED** [review]
The build errors are unrelated.
The build is already red, failing because of a few other existing modules from other modules.
Mergr must be done manually.

**Comment by bodedra** [comment]
@OCA/accounting-maintainers may we merge this?

**Pull request merged**